### ISSUE TITLE
[EN] Support for 'etymid' template

### ISSUE THREAD

**Issue by lasconic** [BoboTiG/ebook-reader-dict#1395]
- Wiktionary page: https://en.wiktionary.org/wiki/early

Wikicode:
```
{{etymid|en|adjective}}
```

Output:
```
<i>Etymid</i> <b>adjective</b>
```

Expected:
Not sure... maybe just
```
<b>adjective</b>
```

---

Model link, if any:

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/tiger-222) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/BoboTiG/ebook-reader-dict/issues/1395">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/BoboTiG/ebook-reader-dict/issues/1395/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/BoboTiG/ebook-reader-dict/issues/1395/pledge.svg">
</picture>
</a>

**Comment by BoboTiG** [comment]
It seems a template to create a sub-section. What if we skip it?

**Comment by BoboTiG** [comment]
Fixed by 53e815db85e81aa667075cb526e0644ef9c9af9b (incorrect sytax in the commit message).
